frida/src/script.rs

```diff
@@ -15,7 +15,8 @@ use crate::{Error, Result};
 
 unsafe extern "C" fn call_on_message<I: ScriptHandler>(
     _script_ptr: *mut _FridaScript,
-    message: *const i8,
+    #[cfg(not(target_arch = "aarch64"))] message: *const i8,
+    #[cfg(target_arch = "aarch64")] message: *const u8,
```

Comment:
@oleavr Is this intended (the message buffer type having a different signedness depending on the architecture)?

Reply:
@meme I think it is an artifact of the usage of `CStr`, which produces differently signed strings.